ui-v2/app/routes/dc/services/show/topology.js

```diff
@@ -0,0 +1,14 @@
+import Route from '@ember/routing/route';
+
+export default Route.extend({
+  model: function() {
+    const parent = this.routeName
+      .split('.')
+      .slice(0, -1)
+      .join('.');
+    return this.modelFor(parent);
+  },
+  setupController: function(controller, model) {
+    controller.setProperties(model);
+  },
+});
```

Comment:
I know you did this before the Outlet loading PR, but now that is merged could you update this Route to use our base Route and add a `this._super` for setupController? 🙇‍♂️